src/util.py

```diff
@@ -260,6 +260,95 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def set_map_boundary(ax, projection, lon_range, lat_range, north_pad=0, south_pad=0, east_pad=0, west_pad=0, res=1):
+    """
+    Utility function to set the boundary of ax to a path that surrounds a
+    given region specified by latitude and longitude coordiantes. This
+    boundary is drawn in the projection coordinates and therefore follows
+    any curves created by the projection. This is currently designed for
+    the Lambert Conformal Projection and North/South Polar Stereographic
+    Projections.
+
+    Args:
+        ax (:class:'matplotlib.axes'):
+            The axes to which the boundary will be applied.
+
+        projection (:class:'cartopy.crs'):
+            The projection of ax.
```

Comment:
I think I can actually remove this argument now. While I was writing the function, I used this to deal with various projections differently. Now that we are limiting this to Lambert Conformal and Polar Stereographic, I can treat those the same way and no longer need the if statement that used `projection`.

Reply:
Sounds good.

Just a quick and early reminder: Please try to finalize the rest of this function as mature as possible (at least for the current two projections of preference). In other words, try to think of close-to-permanent solutions than temporary apporaches as much as possible. This is because, **geocat-viz** serves as an API to **geocat-examples** and is distributed as an independent package via conda. That being said, a newer version of it is going to be generated when this PR is merged, and **geocat-examples** users will update to that version to use this PR's capabilities. So, we should try to design this utility function to make it as mature as possible.

Reply:
Understood. I was originally trying to make this as general of a function as possible, but I quickly realized that it would be too difficult and unnecessary to support so many projections. I was attempting to do too much at once, the code got messy and, I forgot to remove things. Sorry about that. I'll try to finalize this as much as possible before I re-request your review.

Reply:
Totally fine, and we are working on this together. So, we can decide together whenever it is ready for being put into a newer version.